src/pyopticon/_system/_automation_widget.py

```diff
@@ -380,6 +386,17 @@ def _load_automation_file(self): # Open a dialog and load a file for automation
             self.time_to_go_readout.set(str(timedelta(seconds=self.time_to_go)))
         print("Loaded "+filename)
 
+
+    def update(self, event):
+        """When a notification is sent by the publisher (dashboard) the event is checked. If the event has 'New automation file' 
+        in it the existence of an automation file in the Recipe database will be checked, and if it does exist it will be uploaded.
```

Comment:
Recipes are currently used just by us, so we shouldn't put info about them into pyopticon itself. I think this method is fine but we should modify the documentation to remove that reference.